wagtail/admin/views/generic/base.py

```diff
@@ -190,6 +191,7 @@ class BaseListingView(WagtailAdminTemplateMixin, BaseListView):
     index_url_name = None
     index_results_url_name = None
     page_kwarg = "p"
+    paginate_by_kwarg = "p_size"
```

Comment:
```suggestion
    paginate_by_kwarg = "page_size"
```

I believe we generally want more descriptive query parameters (#1064).